frontend/src/components/custom-tools/tool-ide/ToolIde.jsx

```diff
@@ -143,16 +142,14 @@ function ToolIde() {
       return;
     }
 
-    const doc = [...listOfDocs].find((item) => item?.document_id === docId);
-
     const prevSelectedDoc = selectedDoc;
     const data = {
       selectedDoc: doc,
     };
     updateCustomTool(data);
 
     const body = {
-      output: docId,
+      output: doc?.document_id,
```

Comment:
@tahierhussain Eventually I would like us to use ID as capitalized everywhere. Again a longer term fix.
cc @hari-kuriakose

Reply:
@nehabagdia The `document_id` is coming from the backend. It is the field name for the primary key in the document manager model. Since, we're getting the list of documents from the backend, we're using the same list across the prompt studio. 

This is the practice (snackcase) we're following as of now, for all the models in prompt studio. We can discuss this further if any change is required.